mapreduce-pipeline/Compute-Pi-with-lightweight-components-and-minio.ipynb

```diff
@@ -9,8 +9,8 @@
     "# Summary:\n",
```

Comment:
In general, we want these notebooks to include their output so that if someone looks at them without running the notebook themselves, they'll see what should happen.  Please make sure the notebook has been exactly from start to finish once before saving (eg: press restart+run all, then save).  This way we know it works from start to finish without people interacting with it and the users have something to visualize.

In a few cases we have intentionally included cells that will fail.  In that case you can include their error output then continue to run the rest of the notebook